.github/workflows/pr.yml

```diff
@@ -21,6 +21,20 @@ jobs:
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.6.1
 
+      - name: Create Playground Account for Tests
+        run: |
+          playground_account_raw=`curl https://api.prod.us-east-1.warpstream.com/api/v1/signup -X POST -d '{"primary_user_first_name": "playground", "primary_user_last_name": "Account", "tenant_name": "playground-organization-1", "primary_user_email": "playground-email@'$(uuidgen)'", "primary_user_password": "playground-password-'$(uuidgen)'", "is_demo": true}'`
```

Comment:
can you change the email/names/tenant to be "chart-ci" or something so its more obvious in our logging / GTM dashboards?

Reply:
yep